docs/Makefile

```diff
@@ -55,6 +55,7 @@ html:
 	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
+	@echo "View pages at http://127.0.0.1:9991/docs/index.html"
```

Comment:
One thing we may want to mention here is that the index doesn't update automatically for pages that aren't build rebuilt, and they might need to run `rm -rf docs/_build` to update that as well.